README.md

```diff
@@ -26,8 +26,18 @@ cargo generate \
     --destination $PWD/contracts
 ```
 
+## How To Use (without workspace)
+
+Generate a contract template without workspace.
+
+```
+cargo generate \
+    --git https://github.com/Finschia/cw-workspace.git contracts \
+    -d workspace=false
```

Comment:
Can we use without `-d workspace=false`?
If we do not use -d option, ca we use single mode?

Reply:
No, this project is for workspace. Single mode is optional.

Reply:
Could we use placeholder to instead of `-d`?  like after cargo-generate, there's an option (single|workspace).

Reply:
I don't recommend it. It makes this complicated because users must understand the difference between single and workspace.